.vscode/launch.json

```diff
@@ -40,6 +40,7 @@
         "NODE_ICU_DATA": "node_modules/full-icu",
         "TZ": "America/Chicago"
       },
+      "console": "integratedTerminal",
```

Comment:
This was removed (mistakenly, I believe) a few PRs ago. I've re-added it to ensuree that the integrated terminal inside vscode opens when you start this task. Without it, when you click run inside vscode there is no terminal to view the progress of the process.